models/content.rb

```diff
@@ -39,15 +39,6 @@ def author_with_anonymity(attr=nil, attr_when_anonymous=nil)
     end
   end
 
-  def self.flagged
```

Comment:
Why don't we need this?

Reply:
superceded by https://github.com/Stanford-Online/cs_comments_service/pull/3/files#diff-292f8e2aad5b00355c1061f5b0f88e1cR19